### DISCUSSION TITLE
[Projects Beta] Remove Organization from Repository Column

### DISCUSSION THREAD

**Discussion by tn3rb** [community/community#9189] [Projects and Issues]
First off, really excited about the new Projects board, looks great so far. Thanks for all of the effort that has gone into it.

When you add "Repository" as a column to your project, ALL of the values are in the format:

`organization/repository-A`
`organization/repository-B`
`organization/repository-A`
`organization/repository-C`

we unfortunately have a really long organization name, so it ends up looking more like:

`really-long-organization-name/repository-A`
`really-long-organization-name/repository-B`
`really-long-organization-name/repository-A`
`really-long-organization-name/repository-C`

It seems that Projects are specific to a single organization so it's inclusion in the values shown in that column is completely unnecessary and redundant. Valuable screen real estate is taken up by the organization name and any attempt to decrease the column width makes the situation even worse:

`really-long-organization-name/rep`
`really-long-organization-name/rep`
`really-long-organization-name/rep`
`really-long-organization-name/rep`

Since we already know the name of the organization we belong to, and the thing we _really_ only need to see in the repository column is just the name of the repository, it would make sense to remove the organization name from the values in that column so that we are left with:

`repository-A`
`repository-B`
`repository-A`
`repository-C`

which is exactly what we need, no more no less.

Thanks again for the awesome features.

**Comment by Hona** [comment]
This would be pretty cool

**Comment by muizzhaque** [comment]
You can create custom labels for your repositories, which would allow you to display shorter, more concise names in the column. Here’s how you can do it:
Create Custom Labels:
1. Go to your repository settings.
2. Look for the “Labels” section.
3. Create custom labels with shorter names (e.g., “Repo A,” “Repo B,” etc.).

Apply Labels to Repositories:
1. Navigate to your Projects board.
2. Add the custom labels to the relevant repositories.
3. Now, the column will display the custom label names instead of the full repository paths.

Hope it solves this!

**Reply by rantoniuk** [reply]
What do you mean by "Add the custom labels to the relevant repositories." ? There is no [column type](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-board-layout#showing-and-hiding-fields) to shown repository [__topics__](https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization) (ex. labels) on the Project Board, so the repository topics are not visible in that way.

**Comment by rantoniuk** [comment]
Probably the easiest way to implement it would be to add [Custom Repository Properties](https://docs.github.com/en/organizations/managing-organization-settings/managing-custom-properties-for-repositories-in-your-organization) column type to Project Boards?
